snowfall/common.py

```diff
@@ -125,3 +127,115 @@ def save_training_info(
         f.write('best epoch: {}\n'.format(best_epoch))
 
     logging.info('write training info to {}'.format(filename))
+
+
+def read_lexicon(filename: str) -> Dict[str, List[List[str]]]:
```

Comment:
best to use `Path` for paths (or `Pathlike` if the user is likely to pass `str` in anyway)

Reply:
thanks, will change.

Reply:
Resolved by removal.